README.md

```diff
@@ -1,2 +1,30 @@
-# postgres
-Work in progress, not stable, expect force pushes of this repo
+# Autopilot Pattern Postgres
+
+Postgres designed for automated operation using the [Autopilot Pattern](http://autopilotpattern.io/). This repo serves as a blueprint demonstrating the pattern.
```

Comment:
Also, is it worth mentioning it's an extension of an official image and linking out to [said image](https://hub.docker.com/_/postgres/)?